Lib/modulefinder.py

```diff
@@ -72,6 +73,10 @@ def _find_module(name, path=None):
     if spec.loader is importlib.machinery.FrozenImporter:
         return None, None, ("", "", _PY_FROZEN)
 
+    if spec.loader is None:
+        # A namespace package has no loader
```

Comment:
I don't think this comment is necessary.

Reply:
It wasn't obvious to me that namespace packages did not have loaders. That is something I had to dig up for myself. 
I put that comment there to enlighten new maintainers like myself who were never aware of namespace packages.

Do you have a rephrase recommendation?